seacrowd/sea_datasets/malaysia_ai_government/malaysia_ai_government.py

```diff
@@ -0,0 +1,180 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{,
+  author    = {N/A},
+  title     = {No Publication},
+  journal   = {N/A},
+  volume    = {N/A},
+  year      = {N/A},
+  url       = {N/A},
+  doi       = {N/A},
+  biburl    = {N/A},
+  bibsource = {N/A}
+}
+"""
+
+_DATASETNAME = "malaysia_ai_government"
+
+_DESCRIPTION = """\
+This is a dataset containing pdfs scraped from 735 gov.my websites.
+It consists of thousands of the unedited text, a link to the URL where the website was retrieved, and the name of the pdf.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/malaysia-ai/crawl-my-website"
+
+_LANGUAGES = ["zlm"]
+
+
```

Comment:
Nitpick: Just to improve readability of the code, please remove the extra/duplicate blank lines.
"Nitpick" means this not a blocker issue, just a nice to have.